components/resume/themes/DEAFULT_THEME/sections/PersonalInfoSection.tsx

```
@@ -59,7 +63,10 @@ const PersonalInfoSection = ({personalInfoData}: { personalInfoData?: UserInfo }
 					{/* DIVIDER */}
 					<Divider className="mb-1.5" />
 				</div>
-				<p className="text-justify leading-snug text-sm">{personalInfoData?.profile_text}</p>
+				<div 
+					className="text-justify leading-snug text-sm"
+					dangerouslySetInnerHTML={{__html: personalInfoData?.profile_text}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)